### ISSUE TITLE
Call to a member function setCookie() on null

### ISSUE THREAD

**Issue by qibao07** [linuxserver/Heimdall#813]
Happens every few hours.

Delete all cookie, the problem has be fiexd.

So, the key point in cookie, and make [$response->headers] be null.

![image](https://user-images.githubusercontent.com/35797132/160949986-32ad529c-2e48-411e-b2f1-eb7da534d555.png)
![image](https://user-images.githubusercontent.com/35797132/161218941-89fb2d35-589c-4174-8e4b-bdc4284958d6.png)

**Comment by chenweiwen07** [comment]
1. Successfully login to the system using the browser.
2. Restart service.
```
VERSION=2.4.9
NAME=heimdall

docker stop $NAME
docker rm $NAME
docker run -d \
  --name=$NAME \
  -e PUID=$(id -u) \
  -e PGID=$(id -g) \
  -e TZ=Asia/Shanghai \
  -p 57234:80 \
  -v /home/ubuntu/MyDisk2/heimdall-data:/config \
  --restart always \
  linuxserver/heimdall:$VERSION
```
3.  The problem reappeared.

**Comment by KodeStar** [comment]
Do you have 2 copies of Heimdall running?

**Comment by chenweiwen07** [comment]
@KodeStar No, but I have 2 user like this.
![image](https://user-images.githubusercontent.com/35797132/161388230-d3a7a4b5-f9d1-4acf-9bd7-62251f4f8b5b.png)

**Comment by keriati** [comment]
Seems to be duplicate of #379

**Comment by KodeStar** [comment]
Duplicate of #379

**Comment by KodeStar** [comment]
This has been fixed in 2.5.0, see #379 for more details
